main.go

```diff
@@ -59,6 +60,8 @@ func main() {
 	flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&leaderElectionID, "leader-election-id", "a3f45fce.functionmesh.io",
 		"the name of the configmap that leader election will use for holding the leader lock.")
+	flag.StringVar(&leaderElectionNamespace, "leader-election-namespace", "",
+		"the namespace in which the leader election configmap will be created")
 	flag.BoolVar(&enableLeaderElection, "enable-leader-election", true,
```

Comment:
Can we set the `enable-leader-election` to false by default?

Reply:
+1

Reply:
then we need to set that to true manually in k8s cluster environment, is that ok

Reply:
seems already did that, I will update